argo-cd-apps/base/member/infra-deployments/segment-bridge/segment-bridge.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: segment-bridge
+spec:
+  generators:
```

Comment:
seems like you can simplify this to just the `clusters` generator

Reply:
Not too sure about this actually, most of the configs looks the same. does removing the `merge` or `lists` add anything?

Reply:
it would make the code simpler

Reply:
done